pipelines/utility/pipelineExecutor/Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+#!groovy
+
+@Library('piper-library-os') _
+
+// The coordinates of the central pipeline script
+def REPO
+def BRANCH
+def PATH
+
+// In case access to the repository containing the central pipeline
+// script is restricted the credentialsId of the credentials used for
+// accessing the repository needs to be provided below. The corresponding
+// credentials needs to be configured in Jenkins accordingly.
+def CREDENTIALS_ID
+
+node() {
+
+    deleteDir()
+
+    checkout scm
+
+    setupCommonPipelineEnvironment(script: this)
```

Comment:
We have to take the ```configFile``` property (deviant config file) into account. Since we are about to setup the configuration we cannot take this out of the configuration. Currently a deviant config file is not taken into account.
Anyhow the config file used here needs to be the same like the config file in the "central" pipeline.
Of course it would be technically possible to setup the configuration "only" here (and not a second time in the "big" pipeline script. But with that we would introduce additional dependencies between the pre-burner and the "big" pipeline script. ... bad idea, I would say ...